app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,18 @@
+class TasksController < ApplicationController
+
+    def create
+        task = Task.new(task_params)
+        if task.save
+            render json: task
+        else
+            render json: {error: "The task couldn't be added. Try again!"}
+        end
+    end
+
+    private
+
+    def task_params
+        params.require(:task).permit(:description, :is_completed)
```

Comment:
This is going to allow users to set the `is_completed` status when creating a new task. Do we want to give them that ability or just have new tasks default to `is_completed: false`?